src/Microsoft.ML.AutoMLFeaturizers/ToStringTransformer.cs

```diff
@@ -0,0 +1,1763 @@
+    /// </summary>
+    /// <remarks>
+    /// <format type="text/markdown"><![CDATA[
+    ///
+    /// ###  Estimator Characteristics
+    /// |  |  |
+    /// | -- | -- |
+    /// | Does this estimator need to look at the data to train its parameters? | No |
+    /// | Input column data type | Scalar of numeric, boolean, [text](xref:Microsoft.ML.Data.TextDataViewType) |
+    /// | Output column data type | Scalar of [text](xref:Microsoft.ML.Data.TextDataViewType) type. |
+    ///
+    /// The <xref:Microsoft.ML.Transforms.ToStringTransformerEstimator> is a trivial estimator that doesn't need training.
+    /// The resulting <xref:Microsoft.ML.Transforms.ToStringTransformer> converts one or more input columns into its appropriate string representation.
+    ///
+    /// The ToStringTransformer can be applied to one or more columns, in which case it turns each input type into its appropriate string represenation.
+    ///
+    /// ]]>
+    /// </format>
+    /// </remarks>
+    /// <seealso cref="ToStringTransformerExtensionClass.ToStringTransformer(TransformsCatalog, InputOutputColumnPair[])"/>
+    /// <seealso cref="ToStringTransformerExtensionClass.ToStringTransformer(TransformsCatalog, string, string)"/>
+    public sealed class ToStringTransformerEstimator : IEstimator<ToStringTransformer>
+    {
+        private Options _options;
+
+        private readonly IHost _host;
+
+        #region Options
+
+        internal sealed class Column : OneToOneColumn
+        {
+            internal static Column Parse(string str)
+            {
+                Contracts.AssertNonEmpty(str);
+
+                var res = new Column();
+                if (res.TryParse(str))
+                    return res;
+                return null;
+            }
+
+            internal bool TryUnparse(StringBuilder sb)
+            {
+                Contracts.AssertValue(sb);
+                return TryUnparseCore(sb);
+            }
+        }
+
+        internal sealed class Options : TransformInputBase
+        {
+            [Argument(ArgumentType.Multiple | ArgumentType.Required, HelpText = "New column definition (optional form: name:src)",
+                Name = "Column", ShortName = "col", SortOrder = 1)]
+            public Column[] Columns;
+        }
+        #endregion
+
+        // Using this to confirm DLL exists. If does it will just return false since no parameters are being passed.
+        // Once we have a binary dependency on the dll we can remove this code.
+        [DllImport("Featurizers", EntryPoint = "GetErrorInfoString"), SuppressUnmanagedCodeSecurity]
+        private static extern bool CheckIfDllExists(IntPtr error, out IntPtr errorHandleString, out IntPtr errorHandleStringSize);
+
+        public static ToStringTransformerEstimator Create(IHostEnvironment env, string outputColumnName, string inputColumnName)
```

Comment:
These methods shouldn't be `public`. External users should only create the estimators from the Catalog APIs.

Reply:
Changed them all to internal. Fixed in next push.